### PR TITLE
Feature Dashboard PR #3

### PR DESCRIPTION
**Pull request by pzalep1**
This PR adds all of the functionality needed to add, delete, and rearrange the featured objects using the angular drag and drop. This will merge into the Featured Dashboard PR #2 
There is no mobile screenshots because the admin dashboard was never designed for mobile. The screenshots were taken on my 13" Mac.
screencapture-localhost-4200-admin-featured-learning-objects-2020-05-08-13_05_31
screencapture-localhost-4200-admin-featured-learning-objects-2020-05-08-13_06_13

### CODE REVIEW

**Comment by EVogel1999** [comment]
Just to be sure, the "collection" on the card does actually show right?  The first two images doesn't look like they have it.

**Comment by pzalep1** [comment]
> Just to be sure, the "collection" on the card does actually show right? The first two images doesn't look like they have it.

Updated the screenshots. Were taken before I pulled Eni's branch in feature service.

File: src/app/admin/components/draggable-learning-object/draggable-learning-object.component.html [hunk 1]
```diff
@@ -1,26 +1,41 @@
...
+        {{ truncateText(learningObject.name, 45) }}
+      </div>
+      <div class="details">
         <span class="author">
-            {{ learningObject.author.name | titlecase }} |
-            {{ learningObject.author.organization | titlecase}}
-            {{ learningObject.contributors?.length
+          {{ learningObject.author.name | titlecase }} |
+          {{ learningObject.author.organization.trim() | titlecase
+          }}{{
```
Comment (EVogel1999) [inline src/app/admin/components/draggable-learning-object/draggable-learning-object.component.html:26]: Just a typo here.  Don't know why these are on the same line

File: src/entity/learning-object/learning-object.ts [hunk 2]
```diff
@@ -260,6 +260,7 @@ export class LearningObject {
    * @memberof LearningObject
    */
   constructor(object?: Partial<LearningObject>) {
+    console.log(object);
```
Comment (EVogel1999) [inline src/entity/learning-object/learning-object.ts:263]: console log

**Pull request merged**